keylime/tpm2.py

```diff
@@ -30,6 +30,7 @@
 import time
 import zlib
 import yaml
+import codecs
```

Comment:
Same as with `tpm1`, this patch should already be in `master`

Reply:
I am not following, yes its already in master so I performed a merge of master into this branch?

Reply:
It's strange that the diff is showing these changes as being new, since they are already merged into master ...